### PR TITLE
Adding cmd for running stat job via spark-submit

### PR DESCRIPTION
**Pull request by sudhan499**

### CODE REVIEW

File: src/main/java/com/merck/rdf2x/jobs/stats/StatsJob.java [hunk 1]
```diff
@@ -48,6 +48,21 @@
      */
     private final JavaSparkContext sc;
 
+    /**
+     * Subject URI count
+     */
+    private long subjectURICount;
```
Comment (prihoda) [inline src/main/java/com/merck/rdf2x/jobs/stats/StatsJob.java]: Can you use `Long` type and set it to `null` for these three count variables?

File: src/main/java/com/merck/rdf2x/jobs/stats/StatsJob.java [hunk 2]
```diff
@@ -80,12 +95,45 @@ public void run() {
...
+        printStats();
+
+    }
+
+    /**
+     * Prints stats for distinct counts of {@link Quad} properties i.e. subject, predicate and object
+     */
+    private void printStats() {
+        log.info("---------------------------------");
+        log.info("Total Distinct Subject URIs: {}", subjectURICount);
```
Comment (prihoda) [inline src/main/java/com/merck/rdf2x/jobs/stats/StatsJob.java]: Can you print the statistics only for the requested fields? You could set them to `null` and only print it if not `null`. It is a bit confusing to have them set to `0` if they are not requested.

**Review by prihoda — CHANGES_REQUESTED** [review]
Great, thanks for the useful update 👍

Can you please check the comments that I added? Finally, can you update the version in `pom.xml` to <version>0.1.1</version>? Then I will be happy to merge the changes to master and create a new GitHub release.

Thanks!

**Comment by sudhan499** [comment]
Thanks a lot for taking time and reviewing my changes. I made suggested changes, please check.

**Review by prihoda — APPROVED** [review]

**Pull request merged**